tensorflow/compiler/tests/jit_test.py

```diff
@@ -519,6 +521,7 @@ def simpleTest(self, arg0, arg1, global_jit_level):
 
 class LazyCompilationTest(test.TestCase):
 
+  @unittest.skip("test too dependant on XLA compilation protocol")
```

Comment:
This is by design -- the test is testing the XLA compilation protocol. :)

I think the test needs to be adjusted to adapt to whatever the new scheme is,